.github/workflows/collector-builder.yml

```diff
@@ -110,6 +143,24 @@ jobs:
             -e @'${{ github.workspace }}/ansible/secrets.yml' \
             ansible/ci-build-builder.yml
 
+      - name: Build s390x images
```

Comment:
This feels a bit weird, is it easier to build s390x images in a separate step, not as part of the previous one?

Reply:
For now, yeah it's a little simpler, just because we can force the previous step to use localhost only, but this step will use the entire inventory (which is only one VM at the moment)

When I implement full multi-arch native builds I'll hopefully combine these steps